core/src/main/scalajvm/sttp/tapir/TapirFileExtension.scala

```diff
@@ -0,0 +1,15 @@
+package sttp.tapir
+
+import java.nio.file.Path
+import sttp.tapir.internal.TapirFile
+
+trait TapirFileExtension { self: TapirFile =>
+  def toPath: Path = underlying.asInstanceOf[Path]
+  def toFile: java.io.File = toPath.toFile
```

Comment:
do we need both `.toPath` and `.toFile`? Maybe we can just have `.toFile` which would always return the underlying implementation (either jvm or js file). Similarly, we could have `TapirFile.from(...)` with different jvm/js impls. Then there's a slight chance that some code could work with simply `.toFile` and `.from` in a cross-platform way